### PR TITLE
[9.0][MIG] medical pharmacy us

### PR DESCRIPTION
**Pull request by lasley**
This is dependent on
- [x] #77

### CODE REVIEW

**Comment by lasley** [comment]
Rebased and updated to incubator code. This is ready for review.

cc @tedsalmon @kellylougheed

**Review by kellylougheed — APPROVED** [review]

File: medical_pharmacy_us/models/medical_pharmacy.py [hunk 1]
```diff
@@ -1,35 +1,26 @@
...
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2016-2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import fields, models
+from openerp import fields, models, api
```
Comment (tedsalmon) [inline medical_pharmacy_us/models/medical_pharmacy.py]: Import order is incorrect

**Review by tedsalmon — APPROVED** [review]
LGTM, minus minor request 👍

**Comment by woodbrettm** [comment]
@lasley Build was failing from some other things and a merge conflict, so just rebased from 9.0.

**Pull request merged**